msgcntr/messageforums-component-impl/src/java/org/sakaiproject/component/app/messageforums/DiscussionForumServiceImpl.java

```diff
@@ -237,6 +239,9 @@ public String archive(String siteId, Document doc, Stack stack, String archivePa
 	}
 
 	private int appendDiscussionForumElements(String siteId, Document doc, Element messageForumElement, List attachments) {
+		SimpleDateFormat formatter = new SimpleDateFormat(ARCHIVE_DATE_FORMAT);
+		formatter.setTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
why set the time zone here? will the openDate below not have a time zone attached?

if you have to set a time zone, set the server timezone?

Reply:
If one doesn't set a timezone, you get server timezone. Lots of other services export times in UTC so it seemed consistent. But I'm happy to go with server timezone.

Reply:
i'd prefer server time zone. if we need to adjust to UTC later, we can add some manipulation code to kernel

Reply:
Updated to use the default server timezone with explicit timezone in the format string, so:

<discussion_topic anonymous="false" available_close="2023-04-12T08:15+0200" available_open="2023-04-11T14:10+0200" draft="false" id="3836963" locked="false" moderated="false" post_first="false" sort_index="1" subject="First topic">